config/src/main/java/com/epam/aidial/core/config/Config.java

```
@@ -39,4 +54,16 @@ public Deployment selectDeployment(String deploymentId) {
         Assistants assistants = assistant;
         return assistants.getAssistants().get(deploymentId);
     }
+
+
+    public String getCustomApplicationSchema(URI schemaId) {
+        if (schemaId == null) {
```

Could we do just like that?
```
return customApplicationSchemas.get(schemaId);
```

I'm not sure we should throw an exception if schema is not found. Let's a caller to decide

---

fixed